lib/rps_game.rb

```diff
@@ -0,0 +1,29 @@
+class RPSGame
+  
+  attr_reader :player_1_choice, :player_2_choice
+ 
+  def initialize(player_1_choice, player_2_choice = ComputerChoice.new.choice)
+    @player_1_choice = player_1_choice
+    @player_2_choice = player_2_choice
+  end
+
+  def result
```

Comment:
This method is neat but could cause some confusion.  The name, to me, indicates that I will get the result as a string or a 'result object' (whatever that might be), which is true 1/3 of the time but in other cases, it returns a boolean. Methods that return booleans should end with a question mark.

Another way to think of this is that the method will become an interface that you'll use elsewhere.   Which of these do you like the most?

```
if game.result
  'Player one wins'
elseif game.result == false
  'Player two wins'
else
  'Draw'
end
```

or

```
if game.player_1_wins?
  'Player one wins'
elsif game.player_2_wins?
  'Player two wins'
else
  'Draw
end
```
```